test/avocado/run-tests

```diff
@@ -134,7 +134,7 @@ def run_avocado_tests(machine, avocado_tests, print_failed=True, env=[]):
 
 def copy_avocado_logs(machine, title):
     if machine and machine.address:
-        dir = "%s-%s.avocado" % (machine.address, title)
+        dir = os.path.join(testinfra.TEST_DIR, "tmp", "%s-%s.avocado" % (machine.address, title))
```

Comment:
Hi, yes, it is better to have clean names, to see, adressses and what it is. There could be transferred only resulting file. To have copied everything it is easier to debug it, in case something happen, probaby that was reason I implemented it in that way.